brainsets/taxonomy/__init__.py

```
@@ -15,3 +15,16 @@
+_classes = [
+    "Species",
+    "Sex",
+    "Task",
+    "Orientation_8_Classes",
+    "Macaque",
+    "Cre_line",
+    "RecordingTech",
+    "Hemisphere",
+]
+
+__all__ = _classes
```

Ruff (0.15.7)

[error] Invalid format for `__all__`, must be `tuple` or `list`

(PLE0605)